.gitignore

```diff
@@ -1,37 +0,0 @@
-# Object files
```

Comment:
Why you've removed this file?

Reply:
Sorry, it was removed by mistake, it's all because of my ignorance of git. I'll fix it.